android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java

```diff
@@ -194,8 +200,13 @@ public void getSystemBrightness(final Promise promise) {
      */
     @ReactMethod
     public void getAppBrightness(Promise promise) {
-        float brightness = activity.getWindow().getAttributes().screenBrightness;
-        promise.resolve(brightness);
+        Activity activity = getCurrentActivity();
+        if (activity != null) {
```

Comment:
It's a nullable, so I think it should be checked. They specifically flagged it as such, and it's only a getter that subclasses (native modules) call, so I think we are supposed to check.